docs/getting_started/Datus_BI.zh.md

```
@@ -0,0 +1,193 @@
+
+
+#### 3.1.2 在 Datus CLI 中输入：
+
+`.bootstrap-bi`
+
+
+### 3.2 交互流程说明
+
+执行命令后，CLI 会引导你完成以下步骤：
+1. 选择 BI 平台
+  * 例如：Superset / Metabase（取决于已配置的 adaptor，当前只有Superset的实现）
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)